eldritch/items.py

```diff
@@ -238,8 +239,41 @@ def DreadCurse(idx):
   return CombatSpell("Dread Curse", idx, {"magical": 9}, 2, -2, 2)
 
 
-class EnchantWeapon(CombatSpell):
+class BindMonster(CombatSpell):
+  def __init__(self, idx):
+    super().__init__(
+        "Bind Monster", idx, {"magical": float("inf"), "physical": float("inf")}, 2,
+        4, 2,
+    )
+    self.combat_round = None
+
+  def get_required_successes(self, state):  # pylint: disable=unused-argument
+    self.combat_round = state.event_stack[
+        -3
```

Comment:
Is there some better way for the spell to get access to the monster that we're facing?

Reply:
The monster is also an attribute of the CombatChoice. Though as you pointed out, there are some combats that don't have a monster, but I think you would handle that in get_usable_interrupt instead. So I might simplify this to something like
```
choice = state.event_stack[-2]
return choice.monster.toughness(state, choice.character)
```

Reply:
Ah, on attempts to implement this, it turns out that I use self.combat_round in the get_cast_event, so I think I'll just stick with only one dive into the event_stack.